wiki/How_You_Can_Help!/id.md

```diff
@@ -0,0 +1,81 @@
+-   Baca [Kriteria Ranking](/wiki/Ranking_Criteria "Kriteria Ranking") dan kenali apa yang membuat sebuah beatmap dapat berstatus Ranked.
+-   Gabung channel `#modhelp` dan `modreqs` di obrolan dalam game dan bergabunglah dalam percakapan (atau ikuti saja).
+-   Jika Anda memiliki waktu dan dedikasi, buatlah kiriman di forum [Modding Queue](/community/forums/topics/60) dan buat orang berdatangan kepada Anda dengan permintaan.
+
+## Bergabunglah dengan beberapa proyek yang dikelola pengguna dan berkontribusi untuk mewujudkannya (atau menjadikannya lebih baik jika sudah aktif dan berjalan)!
+
+Ada beberapa proposal atau proyek yang sedang dikerjakan, termasuk turnamen, proyek beatmap yang lebih besar, proyek pengembangan aplikasi pihak ketiga dan ide unik lainnya. Silakan hubungi orang-orang yang terlibat dan tunjukkan bahwa Anda tertarik, dan tawarkan bantuan Anda jika Anda memiliki keterampilan yang dapat digunakan.
+
+### Dimulai dari mana:
+
+-   Telusuri forum [Tournaments](/community/forums/55). Turnamen membutuhkan banyak sumber daya manusia untuk mengatur dan mewujudkannya tanpa hambatan, dan Anda dapat membantu dengan hanya duduk sebagai juri untuk pertandingan dalam waktu tertentu.
+-   Periksa apa yang sedang terjadi di forum [General Development](/community/forums/2). Forum ini adalah forum dengan aktivitas rendah di mana [proyek](/community/forums/topics/80027) yang lebih [serius](/community/forums/topics/58855) [terjadi](/community/forums/topics/79354).
+-   Bergabunglah dengan [proyek beatmap](/community/forums/53) dan pelajari betapa menyenangkannya berkolaborasi dengan orang lain!
+-   Mulai proyek Anda sendiri! Jika Anda memiliki ide yang keren, maka cobalah. Jika Anda membutuhkan masukan dari saya, jangan ragu untuk bertanya (meskipun saya lebih suka membantu ketika saya tidak perlu banyak waktu untuk menjawabnya).
+-   Hidupkan kembali [osu!monthly](/community/forums/topics/75017), saya merindukannya :(.
+
+dan yang terakhir, tetapi tidak terlewatkan:
+
+## Hindari berpikiran negatif! Buat orang bahagia!
+
+Setiap hari saya melihat hal negatif di osu!. Orang-orang mengeluh antara satu sama lain, orang-orang mengeluhkan pemain yang curang atau "rank farmers". Orang-orang melihat perubahan dan merengek tentang hal itu. Sementara saya pikir kita merupakan komunitas yang lebih baik dibanding komunitas online lainnya di luar sana - yang turun ke tingkat mulai menghina satu sama lain sepanjang hari (hi youtube!) - masih ada celah yang dapat diperbaiki.
+
+### Dimulai dari mana:
+
+-   Bersikap ramah satu sama lain. Hidup terlalu pendek untuk diperdebatkan.
+-   Don't complain about cheaters. Just [report them and get on with your day](/wiki/Reporting_Bad_Behaviour/Handling_Cheating "Handling Cheating/Foul Behaviour"). They may have taken your top rank, but a simple "!report <username> seems to be cheating" can fix this in minutes to hours.
```

Comment:
wait a min why is this untranslated

Reply:
ffs i forgot to paste that section ;_;
lemme fix